connector-schemas/kafka/connection.json

```diff
@@ -49,6 +49,11 @@
                     "additionalProperties": false
                 }
             ]
+        },
+        "groupId": {
+            "type": "string",
+            "title": "Group Id",
+            "description": "Custom Group ID (Consumer Group) to use"
```

Comment:
This might match the other description a bit better as "Sets the Kafka consumer group id".

It might also be good to add a warning that if you run multiple pipelines with the same consumer group they will not receive all of the data, as it will be shared between them.

Reply:
Altered the description, added caution inside it. Any other way to emphasis it?